services/prometheus/scrape_configs.go

```diff
@@ -464,6 +464,9 @@ func scrapeConfigsForExternalExporter(s *models.MetricsResolutions, params *scra
 		return nil, err
 	}
 
+	// Add a group label.
+	labels["external_group"] = params.service.ExternalGroup
```

Comment:
I think it's better to add it here https://github.com/percona/pmm-managed/blob/4409b0a0a8c69420f82775893e9aa453da911a88/models/service_model.go#L112

Reply:
Moved.